lib/matplotlib/collections.py

```diff
@@ -788,6 +788,7 @@ def update_from(self, other):
         self._facecolors = other._facecolors
         self._linewidths = other._linewidths
         self._linestyles = other._linestyles
+        self._us_linestyles = other._us_linestyles
```

Comment:
no, but I would prefer that be done in a seperate PR (created #7837 to track this).